service_files/phosphor-wait-for-active-passive-target.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Wait for obmc-bmc-active or obmc-bmc-passive target to be started
+After=xyz.openbmc_project.State.BMC.Redundancy.service
+
+[Service]
+ExecStart=/usr/libexec/phosphor-state-manager/wait-for-active-passive-target.sh
+Type=oneshot
+
```

Comment:
"Restart=no" I don't think we'd want to go through the full timeout again with an auto restart on failure?
I was thinking we also may need TimeoutStartSec to set a larger timeout but looks like it's disabled by default with oneshot services.

Reply:
Thanks, will add the Restart=no